README.md

```diff
@@ -1,2 +1,46 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+
+**NOTE**: The procedure will come with downtime while the applications are scaled down for the volume migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1. We assume that you have healthy cluster i.e. `ceph status` shows `health: OK`
+2. Minimum ceph version should be Ceph-CSI supported
+3. Stop the application pods that are consuming the flex volume(s) that need to be converted.
```

Comment:
there is no Rook requirement here?

Reply:
I can add Rook as a requirement here but I thought the requirement for Ceph-CSI is mentioned would be enough
```
This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
```

Reply:
Added.

Reply:
AFAIK this tool cannot work with standalone cephcsi. it will work only if CSI is installed by Rook.

Reply:
okay, I have updated doc with `This guide assumes you have created a Rook-Ceph`